pkg/config/templates/settings.yaml

```diff
@@ -0,0 +1,9 @@
+{{- range $name, $value := .SystemSettings }}
+---
+apiVersion: harvesterhci.io/v1beta1
+kind: Setting
+metadata:
+  annotations:
+    harvesterhci.io/preconfigValue: {{ $value }}
```

Comment:
Could you check if the passed-in `$value` is boolean or number works?
I feel we might need to quote here.

Reply:
Good catch! Although installer doesn't report any error, the annotation would disappear if not quoted.

Reply:
@bk201 Updated. Please help review again, thanks!